core/runtime/src/main/java/io/quarkus/runtime/annotations/RegisterForReflection.java

```diff
@@ -7,7 +7,8 @@
 
 /**
  * Annotation that can be used to force a class to be registered for reflection in native image mode.
- * Note that by default nested classes and interfaces are not registered, unless {@link #ignoreNested()} is set to false.
+ * Note that by default nested classes and interfaces are registered too.
+ * If you have performance concerns, you can use {@link #ignoreNested()} = true.
  * Similarly, by default only the class itself is registered, not the full class hierarchy. This can be changed by setting
```

Comment:
This line needs rephrasing since the hierarchy registration is no longer similar to the nested classes registration.